src/onboarding/Create/Create.js

```diff
@@ -7,13 +7,14 @@ import {
 } from '../../aragonjs-wrapper'
 import { EthereumAddressType } from '../../prop-types'
 import {
-  TRANSACTION_STATUS_ERROR,
+  TRANSACTION_STATUS_UPCOMING,
+  TRANSACTION_STATUS_SIGNING,
   TRANSACTION_STATUS_PENDING,
   TRANSACTION_STATUS_SUCCESS,
-  TRANSACTION_STATUS_UPCOMING,
+  TRANSACTION_STATUS_ERROR,
```

Comment:
It does not seem a big deal, but why breaking the alphabetic sort here?

Reply:
See my answer to your similar question at https://github.com/aragon/aragon/pull/1210#discussion_r362507292